openapi/src/main/java/io/micronaut/openapi/visitor/AbstractOpenApiVisitor.java

```diff
@@ -788,9 +788,8 @@ protected Schema resolveSchema(OpenAPI openAPI, @Nullable Element definingElemen
                     .findFirst()
                     .flatMap(ClassElement::getFirstTypeArgument)
                     .orElse(null);
-            }
```

Comment:
is this change related to https://github.com/micronaut-projects/micronaut-openapi/issues/919 ?